setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
-ignore = F403,E402
-exclude = ve,migrations,local.py
+ignore = F403,E402,F405
+exclude = ve,migrations,local.py,bin,dev.py,production.py,test_settings.py,seaworthy/tests.py,__init__.py
```

Comment:
this will ignore ALL rules in these files. `per-file-ignores` should allow you to set specific rules to ignore for each file. is that not working?